Interface/AddOns/!Colorz/!Colorz.lua

```diff
@@ -1,7 +1,7 @@
 
 -- function SetUpAnimation(frame) CancelAnimations(frame) end
 
-PowerBarColor['MANA'] = {r = 0/255, g = 0.55, b = 1}
+_G.PowerBarColor['MANA'] = {r = 0/255, g = 0.55, b = 1}
```

Comment:
I'm not sure if it is this or the new kill.lua code in the raid frames or a combo of both but I haven't had that error show up since I made the changes. And even if it wasn't what fixed the error setting it this way is the same as using the setglobal function so it is probably a safer way to do it.